docs/platforms/apple/guides/macos/features.mdx

```diff
@@ -1,4 +1,23 @@
-<PlatformContent includePath="getting-started-primer/note" />
+---
+title: Features
+sidebar_order: 1
+description: "Learn about the features of Sentry's Apple SDK for macOS."
+---
+
+Sentry's Apple SDK for macOS enables automatic reporting of errors and exceptions, and identifies performance issues in your application. The minimum supported macOS version is 10.10. The below, is a list of features that are available as part of this SDK.
+
+<Note>
+
+For macOS, a simple [manual setup](/platforms/apple/guides/macos/usage/#capturing-uncaught-exceptions-in-macos) is required for the SDK to report uncaught exceptions.
+
+</Note>
+
+<Note>
+
+The SDK builds a crash report that persists to disk and tries to send the report right after the crash. Since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.
+
+</Note>
```

Comment:
```suggestion
<Note>

The SDK builds a crash report that persists to disk. While it attempts to send the report right after the crash, it may not always work because the environment may be unstable at the time of the crash. If this is the case, the report will be sent upon application start. 

</Note>
```